CONTRIBUTING.md

```diff
@@ -59,14 +59,22 @@ $ charmcraft build
 $ juju deploy ./prometheus-k8s.charm --resource prometheus-image=ubuntu/prometheus:latest
 ```
 
+## Linting
+Flake8 and black linters may be run to check charm and test source code using the
+command
+
+```bash
+tox -e lint
+```
+
 ## Testing
 
 Unit tests are implemented using the Operator Framework test
 [harness](https://ops.readthedocs.io/en/latest/#module-ops.testing). These
 tests may executed by doing
 
 ```bash
-$ ./run_tests
+$ tox -e unit
```

Comment:
You can also clean up requirements.txt because tox now takes care of deps.
I would remove:
- semantic_version (ops takes care of that now)
- coverage
- flake8
- pytest
- pytest-cov

I would keep:
- tox (so you don't pollute the host)
- black (so you can apply changes outside of tox)

Reply:
Removed semantic. But I will like to keep the rest for now. Might consider this in the future when I remove `run_tests`.